src/server/lib/middleware/errorHandler.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from 'express';
+
+export const routeErrorHandler = (
```

Comment:
maybe I should remove this and return the error immediately from src/server/lib/middleware/csrf.ts